.travis.yml

```diff
@@ -25,7 +25,7 @@ env:
 
   matrix:
     - DC_CORE_VERSION="master" SYS_DC_CORE="false"
-    - DC_CORE_VERSION="master" SYS_DC_CORE="true"
+#    - DC_CORE_VERSION="master" SYS_DC_CORE="true"
```

Comment:
Why did you disable this?

Reply:
Because it's not working.

Reply:
This explains things ^^ It's not very important, we only need it if we want to use a system wide installed libdeltachat. As there's currently no such a package using this, i'm fine with breaking it.

Reply:
We first need to make it happen in rust core, i.e. there's no `make install` step there yet to install the library system wide. So I figured we scrap it in desktop until that is working properly.

Reply:
Yup i'm fine with this :) Let's merge!